src/solve.jl

```diff
@@ -313,3 +251,33 @@ project(reach_sol::AbstractSolution) = project(reach_sol.Xk, reach_sol.options)
 
 project(Rsets::Vector{<:LazySet}, options::Pair{Symbol,<:Any}...) =
     project(Rsets, Options(Dict{Symbol,Any}(options)))
+
+# ===========================================================================
+# Bogomolov, Foretes, Frehse, Podolski, Schillic, Viry 2018
```

Comment:
Bogomolov, Forets, Frehse, Podelski, Schilling, Viry 2018

Reply:
that'a a typo in my own name 😄